core/test/fixtures/config/lighthouse-plugin-config-helper/plugin-config-helper.js

```diff
@@ -23,3 +22,5 @@ module.exports = {
     ],
   },
 };
+
+throw new Error('This file is never actually used. If that changes, just delete this line.');
```

Comment:
Is this supposed to be an example plugin or something? Seems like we can remove in favor of our docs example.

Reply:
Perhaps! But for now, I am just documenting this as it took me a bit to confirm.